packages/core/src/components/Card/card.scss

```diff
@@ -0,0 +1,61 @@
+.cds-card {
+  display: flex;
+  flex-direction: column;
+  background-color: var(--cds-card-background-color);
+  color: var(--cds-card-body-color);
+  border: var(--cds-card-border-color) var(--cds-card-border-width);
+  border-radius: var(--cds-card-border-radius);
+  box-shadow: var(--cds-card-shadow);
+
+  &__image {
+    width: 100%;
+    height: auto;
+    object-fit: cover;
+    border-top-left-radius: var(--cds-card-border-radius);
+    border-top-right-radius: var(--cds-card-border-radius);
+  }
+
+  &__image--only {
+    border-radius: var(--cds-card-border-radius);
```

Comment:
It doesn’t look like this value is applied to the image when only an `imagePath` is supplied—meaning only the top left/top right corners are rounded while the bottom are square:

<img width="983" alt="image" src="https://github.com/compassion-global-experience/compassion-design-system/assets/189007/5fc12e9a-a94a-4b08-8c86-d2cdbb566e42">